src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+
     public String getRandomColor() {
-        return null;
+        Color[] colors = Color.values();
+        Color randomColor = colors[new Random().nextInt(colors.length)];
+        return randomColor.toString();
```

Comment:
read checklist about using `.toString()` or `.name()` in enums and make corresponding fix